flixel/system/render/blit/FlxBlitView.hx

```diff
@@ -0,0 +1,435 @@
+		if (camera.width != buffer.width || camera.height != buffer.height)
+		{
+			var oldBuffer:FlxGraphic = screen.graphic;
+			buffer = new BitmapData(camera.width, camera.height, true, 0);
+			
+			screen.graphic = FlxGraphic.fromBitmapData(buffer, false, null, false);
+			oldBuffer.destroy();
+			screen.origin.set();
+			_flashBitmap.bitmapData = buffer;
+			_flashRect.width = camera.width;
+			_flashRect.height = camera.height;
+			_fill = FlxDestroyUtil.dispose(_fill);
+			_fill = new BitmapData(camera.width, camera.height, true, FlxColor.TRANSPARENT);
+			
+		}
+		
+		regen = false;
+	}
+	
+	override public function updateScale():Void 
+	{
+		_flashBitmap.scaleX = camera.totalScaleX;
+		_flashBitmap.scaleY = camera.totalScaleY;
+	}
+	
+	override public function fill(Color:FlxColor, BlendAlpha:Bool = true, FxAlpha:Float = 1.0):Void 
+	{
+		if (BlendAlpha)
+		{
+			_fill.fillRect(_flashRect, Color);
+			buffer.copyPixels(_fill, _flashRect, _flashPoint, null, null, BlendAlpha);
+		}
+		else
+		{
+			buffer.fillRect(_flashRect, Color);
+		}
+	}
+	
+	override public function drawFX(FxColor:FlxColor, FxAlpha:Float = 1.0):Void 
+	{
+		var alphaComponent:Float = FxColor.alpha;
+		fill((Std.int(((alphaComponent <= 0) ? 0xff : alphaComponent) * FxAlpha) << 24) + (FxColor & 0x00ffffff));
+	}
+	
+	override public function lock(useBufferLocking:Bool):Void 
+	{
+		checkResize();
+		if (useBufferLocking)
+			buffer.lock();
+		
+		fill(camera.bgColor, camera.useBgAlphaBlending);
+		screen.dirty = true;
+	}
+	
+	override public function offsetView(X:Float, Y:Float):Void 
+	{
+		flashSprite.x += X;
+		flashSprite.y += Y;
+	}
+	
+	override private function set_color(Color:FlxColor):FlxColor 
+	{
+		if (_flashBitmap == null)
+			return Color;
+		
+		var colorTransform:ColorTransform = _flashBitmap.transform.colorTransform;
+		colorTransform.redMultiplier = Color.redFloat;
+		colorTransform.greenMultiplier = Color.greenFloat;
+		colorTransform.blueMultiplier = Color.blueFloat;
+		_flashBitmap.transform.colorTransform = colorTransform;
+		return Color;
+	}
+	
+	override private function set_alpha(Alpha:Float):Float 
+	{
+		if (_flashBitmap == null)
+			return Alpha;
+		
+		var colorTransform:ColorTransform = _flashBitmap.transform.colorTransform;
+		colorTransform.alphaMultiplier = Alpha;
+		_flashBitmap.transform.colorTransform = colorTransform;
+		return Alpha;
+	}
+	
+	override public function unlock(useBufferLocking:Bool):Void 
+	{
+		if (useBufferLocking)
+			buffer.unlock();
+		
+		screen.dirty = true;
+	}
+	
+	override public function beginDrawDebug():Graphics 
+	{
+		FlxSpriteUtil.flashGfx.clear();
+		return FlxSpriteUtil.flashGfx;
+	}
+	
+	override public function endDrawDebug():Void 
+	{
+		buffer.draw(FlxSpriteUtil.flashGfxSprite);
+	}
+	
+	override private function set_angle(Angle:Float):Float 
+	{
+		if (flashSprite != null)
+			flashSprite.rotation = Angle;
+		
+		return Angle;
+	}
+	
+	override public function set_smoothing(Smoothing:Bool):Bool 
```

Comment:
of course!
But it's strange, because this method is private in super class. I didn't know that you could change method visibility modifier in subclasses